javascript/javascript1/week4/voiceAssistant.js

```diff
@@ -0,0 +1,133 @@
+let myName = "";
+const toDoList = [];
+
+function getReply(command) 
+{
+    let x = command.toLowerCase();
+    const start = x.indexOf("is");
+    const name = x.substring(start + 2).trim();
+if (x.includes("my name is")) 
+{
+    if (name == "") 
+    {
+        console.log('Please enter your name')
+        myName = prompt('What is your name?')
+        return `Nice to meet you ${myName}`;
+    } 
+    else {
+    myName = name
+    return `Nice to meet you ${myName}`;
+    }
+} 
+else if (x.includes("what is my name")) 
+{
+    return `Your name is ${myName}`;
+} 
+else if (x.includes("to my todo")) 
+{
+    const start = x.indexOf("add");
```

Comment:
You can't be sure there's "add" in the string. You're only checking for "to my todo".